File: sheet.ts
interface Celldata {
	object?: 'ABSOLUTE' | 'RELATIVE' | 'ACTION';
	color: string;
	pointer: Number | String;
	content: string;
}

export interface Layerdata {
	projectCode: string;
	layerName: string;
	layerIndex: Number;
	layerNumber: string;
	stt: Date | null;
	end: Date | null;
	aud: string;
	iss: string;
	isc: string;
	body: Array<Array<Celldata>>;
}

File: sheet.py
from datetime import date

class Celldata:
    object: str
    color: str
    pointer: str
    content: str

class Layerdata:
    projectCode: str
    layerName: str
    layerIndex: int
    layerNumber: str
    stt: date
    end: date
    iss: str
    isc: str
    body: list